docker/apollo-server/schemaFetcher.ts

```
@@ -1,4 +1,4 @@
-import * as fs from 'node:fs';
+﻿import * as fs from 'node:fs';
```

ESLint

[error] Unexpected Unicode BOM (Byte Order Mark).

(unicode-bom)